src/components/atoms/StartAgainButton.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ActionButton } from "../atoms/ActionButton";
+
+/**
+ * This component will add a start again button
+ */
+
+export function StartAgainButton(props) {
```

Comment:
hmmm not entirely sure what the point of this component is. perhaps modifying the ActionButton component is enough

Reply:
That's true , I will just modify the action button instead. Thanks Omar